Antidote/ActiveCallViewController.m

```diff
@@ -409,11 +411,27 @@ - (void)setupIncomingCallViewForFriend:(NSString *)nickname
     }];
 }
 
+- (void)startTableViewRefreshTimer
+{
+    self.tableViewRefreshTimer = [NSTimer scheduledTimerWithTimeInterval:1.0
+                                                                  target:self
+                                                                selector:@selector(reloadPausedCalls)
+                                                                userInfo:nil
+                                                                 repeats:YES];
+}
+
 #pragma mark - Call Menu
 
 - (void)callMenuButtonPressed
 {
     self.tableViewOfPausedCalls.hidden = ! self.tableViewOfPausedCalls.hidden;
+
+    if (! self.tableViewOfPausedCalls.hidden) {
+        [self startTableViewRefreshTimer];
+    }
+    else {
+        [self.tableViewRefreshTimer invalidate];
+    }
```

Comment:
It is better to avoid negation, this makes code more readable.
```
if (self.tableViewOfPausedCalls.hidden) {
    [self.tableViewRefreshTimer invalidate];
}
else {
    [self startTableViewRefreshTimer];
}
```